tests/integration/dom/scroll-to-test.js

```diff
@@ -0,0 +1,100 @@
+import { module, test } from 'qunit';
+import hbs from 'htmlbars-inline-precompile';
+import {
+  render,
+  scrollTo,
+  setupContext,
+  setupRenderingContext,
+  teardownContext,
+} from '@ember/test-helpers';
+
+module('DOM Helper: scroll-to', function(hooks) {
+  hooks.beforeEach(async function() {
+    await setupContext(this);
```

Comment:
These style tests only work from Ember 2.4+, if you checkout the other tests you'll see a guard somewhere in the root of the module.

Reply:
https://github.com/emberjs/ember-test-helpers/blob/fdcf31ab04b908e4561eeea102fc07a6f483d640/tests/unit/dom/click-test.js#L8-L10